courses/free-courses-en.md

```diff
@@ -1053,7 +1053,6 @@
 
 * [Beginner's Typescript](https://www.totaltypescript.com/tutorials/beginners-typescript) - Matt Pocock
 * [ES6 and Typescript Tutorial](https://www.youtube.com/playlist?list=PLC3y8-rFHvwhI0V5mE9Vu6Nm-nap8EcjV) - Codevolution, Vishwas Gopinath
-* [Introduction to TypeScript](https://www.udemy.com/typescript/) - Daniel Stern (Udemy)
```

Comment:
not sure why this is here

Reply:
Sorry, I made a mistake here, I will correct this file.